pygeoapi/provider/postgresql.py

```diff
@@ -87,18 +90,18 @@ def __init__(self, conn_dic, table, context="query"):
         self.context = context
         self.columns = None
         self.conn = None
-        self.schema = None
 
     def __enter__(self):
         try:
-            self.schema = self.conn_dic.pop('schema', None)
-            if self.schema == 'public' or self.schema is None:
+            self.search_path = self.conn_dic.pop('search_path', None)
+            if self.search_path is ['public'] or self.search_path is None:
                 pass
             else:
-                self.conn_dic["options"] = '-c search_path={}'.format(
-                    self.schema)
-                LOGGER.debug('Using schema {} as search path'.format(
-                    self.schema))
+                if "public" not in self.search_path:
+                    self.search_path = self.search_path + ["publc"]
```

Comment:
Typo: `publc`; and do we always want to add this schema? Why?

Reply:
I think `['public']` should be the default if the property is completely unspecified, but I don't think it needs to be enforced if it's explicitly left out from the configuration.

Reply:
Agree (do not add `public` if missing in `search_path`) .